src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -37,6 +38,94 @@
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def get_crop_region(mask_image: PIL.Image.Image, width: int, height: int, pad=0):
```

Comment:
Should we maybe move this also into `image_processor` and make it a function of the image processor object?

Reply:
ok to move there, but I'm i favor not to: this is a super specific function I don't think will be used for another purpose... also need the image to be a "mask" for it to work

Reply:
Ok to use # Copied from now, but we'll already have to use it for all our inpaint pipelines (SD, SDXL, SD + ControlNet, ...)

Reply:
good point, I will move to the image processor then 😅